File: sqry-lang-csharp/tests/fixtures/csharp/localvars/basic.cs
namespace LocalVars
{
    class BasicVars
    {
        void SimpleVar()
        {
            int x = 10;
            int y = x + 1;
            Console.WriteLine(y);
        }

        void VarKeyword()
        {
            var count = 42;
            Console.WriteLine(count);
        }

        void MultipleVars()
        {
            int a = 1, b = 2;
            int c = a + b;
            Console.WriteLine(c);
        }

        void ParamRef(string name, int age)
        {
            var result = name;
            Console.WriteLine(result);
            Console.WriteLine(age);
        }
    }
}

File: sqry-lang-python/tests/fixtures/localvars/basic.py
# Basic: simple assignment, parameters, constants

def simple_var():
    x = 10
    y = x + 1
    return y

def const_binding():
    count = 42
    result = count + 1
    return result

def mutable_var():
    x = 10
    x = x + 1
    return x

def param_ref(name, age):
    result = name
    total = age + 1
    return result
